viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1631,6 +1682,42 @@ export class Cognite3DViewer {
   ): Promise<CameraManagerCallbackData> {
     const intersection = await this.intersectModels(offsetX, offsetY, { asyncCADIntersection: false });
 
+    const normalizedCoords = getNormalizedPixelCoordinates(this.domElement, offsetX, offsetY);
+
+    const result: CameraManagerCallbackData = {
+      intersection,
+      modelsBoundingBox: this._updateNearAndFarPlaneBuffers.combinedBbox,
+      pickedBoundingBox: undefined
+    };
+
+    let closestDistanceToCamera = result.intersection?.distanceToCamera ?? undefined;
```

Comment:
You don't need to add `?? undefined`, the `result.intersection?.distanceToCamera` will already be undefined if `result.intersection` is undefined or `result.intersection.distanceToCamera` is undefined.